app/src/main/java/com/courseproject/tindar/entities/Account.java

```diff
@@ -0,0 +1,317 @@
+package com.courseproject.tindar.entities;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+@SuppressWarnings("FieldMayBeFinal")
+
+public class Account {
+    private boolean isActiveStatus;
+    private final String id;
+    private String email;
+    private String password;
+    private String displayName;
+    private String firstName;
+    private String lastName;
+    private String aboutMe;
+    private String profilePictureLink;
+    private ArrayList<String> matchList;
+    private ArrayList<String> likeList;
+    private ArrayList<String> blockList;
+    private ArrayList<String> recentlyViewed;
+    private Date birthdate;
+    private String location;
+    private String gender;
+    private Filters filters;
+    /**
+     * Initializes an Account object which is used to store information
+     * about a user.
+     *
+     * @param id the back-end user id associated with this account
+     * @param email the email associated with this account
+     * @param password the password used to log in to this account
+     * @param displayName the username chosen to represent this account
+     * @param firstName the account owner's first name
+     * @param lastName the account owner's last name
+     */
+    Account(String id, String email, String password,
+                   String displayName, String firstName, String lastName) {
+        this.isActiveStatus = true;
+        this.id = id;
+        this.email = email;
+        this.password = password;
+        this.displayName = displayName;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.aboutMe = "";
+        this.profilePictureLink = "";
+        this.matchList = new ArrayList<>();
+        this.likeList = new ArrayList<>();
+        this.blockList = new ArrayList<>();
+        this.recentlyViewed = new ArrayList<>();
+        this.birthdate = null;
+        this.location = null;
+        this.gender = null;
+        this.filters = null;
+    }
+    /** Returns the status of an account in the form of a String.
+     *
+     * @return the status of the account
+     */
+    String getAccountStatus() {
```

Comment:
I just noticed we don't have access modifiers for the methods here. Don't we want to have it public?